hooks/_common.sh

```diff
@@ -55,6 +55,44 @@ function common::parse_cmdline {
   done
 }
 
+#######################################################################
+# Expand environment variables definition into their values in '--args'.
+# Support expansion only for ${ENV_VAR} vars, not $ENV_VAR.
+# Globals (modify):
+#   ARGS (array) arguments that configure wrapped tool behavior
+#######################################################################
+function common::parse_and_export_env_vars {
+  local -r len=${#ARGS[@]}
+  local arg
+
+  for ((i = 0; i < len; i++)); do
+    arg="${ARGS[$i]}"
+
+    # Repeat until all env vars will be expanded
+    while true; do
+      # Check if at least 1 env var exists in `$arg`
+      # shellcheck disable=SC2016 # '${' should not be expanded
+      if [[ "$arg" =~ .*'${'[A-Z_][A-Z0-9_]+?'}'.* ]]; then
+        # Get `ENV_VAR` from `.*${ENV_VAR}.*`
+        env_var_name=${arg#*$\{}
+        env_var_name=${env_var_name%%\}*}
+        env_var_value="${!env_var_name}"
+        # shellcheck disable=SC2016 # '${' should not be expanded
+        common::colorify "green" 'Found ${'"$env_var_name"'} in:        '"'$arg'"
+        # Replace env var name with its value.
+        # `$arg` will be checked in `if` conditional, `$ARGS` will be used in the next functions.
+        # shellcheck disable=SC2016 # '${' should not be expanded
+        arg=${arg/'${'$env_var_name'}'/$env_var_value}
+        ARGS[$i]=$arg
```

Comment:
Tempting fate once more 2:
```suggestion
        ARGS[$arg_idx]=$arg
```